assets/src/dashboard/components/button/index.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types'; // import styled from 'styled-components';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { BUTTON_TYPES } from '../../constants';
+
+const StyledButton = styled.button`
+  align-items: center;
+  background-color: ${({ theme, type }) =>
+    type === BUTTON_TYPES.PRIMARY ? theme.colors.bluePrimary : 'transparent'};
```

Comment:
Not a strong opinion but I think I prefer making a `const PrimaryButton = styled(StyledButton)` and doing the switching in javascript

Reply:
sounds good!